.changeset/grey-triangles-shout.md

```diff
@@ -0,0 +1,68 @@
+---
+"@fluidframework/tree": minor
+---
+---
+section: tree
+---
+
+Allow associating metadata with Node Schema
+
+Users of TreeView can now specify metadata when creating Node Schema.
+This includes system-understood metadata, e.g., `description`.
+
+Example:
+
+```typescript
+
+class Point extends schemaFactory.object("Point", {
+	x: schemaFactory.required(schemaFactory.number),
+	y: schemaFactory.required(schemaFactory.number),
+},
+{
+	metadata: { description: "A point in 2D space" }
+}) {}
+
+```
+
+Functionality like the experimental conversion of Tree Schema to [JSON Schema](https://json-schema.org/) (`getJsonSchema`) leverages such system-understood metadata to generate useful information.
```

Comment:
```suggestion
Functionality like the experimental conversion of Tree Schema to [JSON Schema](https://json-schema.org/) (`getJsonSchema`) uses such system-understood metadata to generate useful information.
```

Reply:
I guess I'm not sure why the latter is better here. Both are "correct", where my wording is technically a bit more specific. I don't really have a strong opinion either way, but I don't see the benefit in the change.